### DISCUSSION TITLE
unequip item

### DISCUSSION THREAD

**Discussion by omggamerR** [runelite/runelite#17180] [Ideas]
setting pre selected spaces in inventory to click when empty to remove item e.g (helmet take-off)

i think that would be awesome

**Comment by Nightfirecat** [comment]
We cannot add that [as it violates Jagex's Third Party Client Guidelines](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1) regarding adding menu entries which send requests to the server.

**Reply by omggamerR** [reply]
very efficient thank you
